signalflow/client.go

```diff
@@ -0,0 +1,307 @@
+package signalflow
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"net/url"
+	"path"
+	"sync"
+	"sync/atomic"
+	"time"
+
+	"github.com/gorilla/websocket"
+	"github.com/signalfx/signalfx-go/signalflow/messages"
+)
+
+// Client for SignalFlow via websockets (SSE is not currently supported).
+type Client struct {
+	// Access token for the org
```

Comment:
Docs suggest that this can be either an org token or a user access token.

> To use the SignalFlow API, you need to have an org token or a user access token.


https://developers.signalfx.com/signalflow_analytics/signalflow_overview.html

Reply:
Yes, I tested and it accepts either type.  Will clarify.